app/models/report.rb

```diff
@@ -44,36 +23,13 @@ class Report < TvarkauVilniuDbBase
               less_than_or_equal_to: 180
             }
 
-  validate :report_type_validation
-
-  belongs_to :report_type, class_name: 'ReportType', foreign_key: 'PROBLEM_TYPE_ID'
-  belongs_to :status_type, class_name: 'StatusType', foreign_key: 'STATUS_ID'
-  belongs_to :main_cfg, class_name: 'MainCfg', foreign_key: 'CITY_ID'
+  after_initialize :generate_ref_no
```

Comment:
This will be generated and updated by `@vilys` integration service so it should be `nil` when creating report.